modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1194,6 +1194,27 @@ CV__DNN_INLINE_NS_BEGIN
                              CV_OUT std::vector<int>& indices,
                              const float eta = 1.f, const int top_k = 0);
 
+    /** @brief Performs batched non maximum suppression on given boxes and corresponding scores across different classes.
+
+     * @param bboxes a set of bounding boxes to apply NMS.
+     * @param scores a set of corresponding confidences.
+     * @param idxs a set of corresponding classes.
+     * @param score_threshold a threshold used to filter boxes by score.
+     * @param nms_threshold a threshold used in non maximum suppression.
+     * @param indices the kept indices of bboxes after NMS.
+     * @param eta a coefficient in adaptive threshold formula: \f$nms\_threshold_{i+1}=eta\cdot nms\_threshold_i\f$.
+     * @param top_k if `>0`, keep at most @p top_k picked indices.
+     */
+    CV_EXPORTS void BatchedNMSBoxes(const std::vector<Rect>& bboxes, const std::vector<float>& scores, const std::vector<int>& idxs,
```

Comment:
What is about using the suffix? (mode or function variant)

`NMSBoxesBatch()`

Reply:
Replaced with `NMSBoxesBatched` to be in sync with `NMSBoxesRotated`.

Reply:
Replaced with `NMSBoxesBatched` to be in sync with `NMSBoxesRotated`.